common/reviews/api/rush-lib.api.ts

```diff
@@ -264,6 +268,17 @@ module RushConstants {
 
 }
 
+// (undocumented)
```

Comment:
In general, don't introduce any new definitions to api.ts that are missing documentation

---
In reply to: [116885084](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r116885084) [](ancestors = 116885084)